### PR TITLE
Set cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)

### PR DESCRIPTION
**Pull request by agaida**
Removed the cmake version checks and just set it

### CODE REVIEW

File: examples/use-qtxdg/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,4 @@
 project(use-qtxdg)
-cmake_minimum_required(VERSION 2.8.12)
```
Comment (palinek) [inline examples/use-qtxdg/CMakeLists.txt]: Why did you remove it here? It should (must) just be the first statement in the main CMakeLists.txt file... and this an example of project using the xdg, isn't it?

Comment (agaida) [reply examples/use-qtxdg/CMakeLists.txt]: Good point, i saw it as part of this project only.
